Networking/Networking/Remote/LeaderboardsRemote.swift

```diff
@@ -72,5 +82,6 @@ private extension LeaderboardsRemote {
         static let after: String    = "after"
         static let before: String   = "before"
         static let quantity: String = "per_page"
+        static let forceRefresh: String = "force_cache_refresh"
```

Comment:
```suggestion
        static let forceRefresh = "force_cache_refresh"
```

Reply:
I was just following the other constants in the same enum, so I removed all in https://github.com/woocommerce/woocommerce-ios/pull/7622/commits/373269051a4d18c7cd0e8cd192e86c6faf88f9f4